Dockerfile

```diff
@@ -39,7 +43,8 @@ COPY containerbuild/bin/entrypoint /home/${USER_NAME}/bin
 
 RUN chown -R `id -u`:0 /home/${USER_NAME}/bin && chmod -R 755 /home/${USER_NAME}/bin
 USER ${USER_UID}:0
-ENTRYPOINT "/home/srt/bin/entrypoint" "--broker-uri" "$BROKER_URI" "-v" "$LOG_LEVEL" "--sleep-cycle" "$SLEEP_CYCLE"
+ENTRYPOINT /home/srt/bin/entrypoint --broker-uri ${BROKER_URI} --sleep-cycle ${SLEEP_CYCLE}
```

Comment:
@cybtachyon This looks like a very intentional change for a potentially good reason, but I'm not seeing it :) What was the reasoning here to move the -v and LOG_LEVEL ENV var usage to be passed via CMD instead of the ENTRYPOINT string?

Reply:
The removal of the quotes prevents string errors with CPP, the moving of LOG_LEVEL to CMD makes it easier to change log levels with a single docker exec.